### PR TITLE
[CX-1878]: Data App Jobs page

### PR DESCRIPTION
**Pull request by anatoly-scherbakov**
Implemented **Tracking Jobs** page for data apps.

### CODE REVIEW

**Review by fpopa — APPROVED** [review]

**Comment by leoebfolsom** [comment]
@anatoly-scherbakov this looks good to me, but I noticed it's being merged into `anatoly-cx-1878-tableau-customer-docs`. Is that intentional? I'll approve with the assumption that yes, it is intentional, and then I will assume also that another PR will eventually appear for review to merge `anatoly-cx-1878-tableau-customer-docs` into `main`. 

Please LMK if I've misunderstood -- thank you!

File: docs/integrations/bi_data_apps/tracking_jobs.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+---
+sidebar_position: 4
+title: Tracking Jobs
+description: ""
+#pagination_prev: deployment_testing/team_cloud/getting_started_for_customers/dbt
+#pagination_next: guides
+---
+
+When a new data integration has been created, we can track whether the sync has been completed and whether it was successful. For that, navigate to **:wrench: Jobs** tab in the left sidebar.
```
Comment (leoebfolsom) [inline docs/integrations/bi_data_apps/tracking_jobs.md:9]: "When a new data app integration has been created"

(Note the addition of "app")

Comment (leoebfolsom) [reply docs/integrations/bi_data_apps/tracking_jobs.md:9]: "For that, navigate to the **:wrench: Jobs** tab in the left sidebar."

(note the addition of "the")

File: docs/integrations/bi_data_apps/tracking_jobs.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+![](/img/data_app_jobs.png)
+
+Your **Search** and **Lineage** for a newly created Data App integration are ready when there is a job marked as `Done` for your integration on this screen.
```
Comment (leoebfolsom) [inline docs/integrations/bi_data_apps/tracking_jobs.md:13]: I would lowercase Data App to be consistent with the rest of this PR (and hopefully the rest of the docs, other than titles and headers, but I haven't reviewed every line)

**Review by leoebfolsom — APPROVED** [review]
Approved with minor suggestions, thanks!

**Comment by anatoly-scherbakov** [comment]
@leoebfolsom yes, this PR can only be merged after the previous one in the stack, which is https://github.com/datafold/datafold-docs/pull/230, is merged. Please check it out when you have a moment :) You've been automatically asked for review as a code owner.

**Comment by leoebfolsom** [comment]
@anatoly-scherbakov I reviewed and merged #230 (looks great!), but now there are merge conflicts here -- please see if you can easily resolve those and then tag me if you'd like me to review this one.

**Pull request merged**